### PR TITLE
Security Wording Updates

### PR DESCRIPTION
**Pull request by basitshipyard**
Here's my pass on updating the wording on some of the security stuff.

### CODE REVIEW

File: background/keeping-your-data-secure.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ In other words, we make sure the pipes are working effectively, but we'll never
 Our application has been built with security in mind at every step.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:11]: On line 3, "of storing any underlying" or "of storing any of the underlying"

File: background/keeping-your-data-secure.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ In other words, we make sure the pipes are working effectively, but we'll never
 Our application has been built with security in mind at every step. 
 
 * All infrastructure is built on AWS, enabling us to have the highest level of [cloud security](https://aws.amazon.com/security/).
-* User passwords are hashed using Bcrypt with strong salts.
-* All data stored in our databases is encrypted at rest.
-* All Blueprint and Vessel specific configurations are additionally encrypted at the application level.
-* All external network traffic from the application is encrypted with TLS.
+* User passwords are stored using the industry leading Blowfish-based Bcrypt encryption algorithm with extremely strong salts.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:14]: Bcrypt isn't an encryption algorithm. Just say Bcrypt algorithm.

File: background/keeping-your-data-secure.md [hunk 3]
```diff
@@ -11,10 +11,10 @@ In other words, we make sure the pipes are working effectively, but we'll never
...
+* All data stored and processed in our databases is encrypted while in transit and at rest using RSAES-OAEP-SHA-256 asymmetric algorithms.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:15]: RSAES-OAEP-SHA-256 is a key transport algorithm. That's not what RDS is using behind the scenes. Did you pull this from AWS docs somewhere, @basitshipyard ?

File: background/keeping-your-data-secure.md [hunk 4]
```diff
@@ -11,10 +11,10 @@ In other words, we make sure the pipes are working effectively, but we'll never
 Our application has been built with security in mind at every step. 
 
 * All infrastructure is built on AWS, enabling us to have the highest level of [cloud security](https://aws.amazon.com/security/).
-* User passwords are hashed using Bcrypt with strong salts.
-* All data stored in our databases is encrypted at rest.
-* All Blueprint and Vessel specific configurations are additionally encrypted at the application level.
-* All external network traffic from the application is encrypted with TLS.
+* User passwords are stored using the industry leading Blowfish-based Bcrypt encryption algorithm with extremely strong salts.
+* All data stored and processed in our databases is encrypted while in transit and at rest using RSAES-OAEP-SHA-256 asymmetric algorithms. 
+* All Blueprint and Vessel specific configurations are encrypted at the application level with RSA-4096 bit key types.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:16]: with AES-256-GCM.

File: background/keeping-your-data-secure.md [hunk 5]
```diff
@@ -26,22 +26,22 @@ All other data stored by Shipyard is meta-data generated by our platform or fron
 
 **Code**
 
-All code provided to us through either uploading the code or writing the code directly in the UI gets uploaded directly to S3. If code for a specific Vessel is ever updated, the old version of the code gets routinely wiped every 7 days.
+All code provided to us through either uploading the code or writing the code directly in the UI gets uploaded to S3 and is stored using AES-256 encryption. If code for a specific Vessel is ever updated, the old version of the code gets routinely wiped every 7 days.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:29]: > If code for a specific Vessel is ever updated, the old version of the code gets routinely wiped every 7 days.

This isn't true. It stays around forever in S3.

File: background/keeping-your-data-secure.md [hunk 6]
```diff
@@ -26,22 +26,22 @@ All other data stored by Shipyard is meta-data generated by our platform or fron
...
 
 **Credentials**
 
-All credentials provided to the application as Environment Variables are both encrypted at rest and at the application level.
+All credentials provided to the application as Environment Variables are both encrypted at rest and at the application level while in transit.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:33]: All credentials, as part of the Blueprint and Vessel configurations, are encrypted in transit, by the application using AES-256-GCM, and at rest.

File: background/keeping-your-data-secure.md [hunk 7]
```diff
@@ -53,14 +53,14 @@ When it comes to storing and organizing large datasets, we recommend using a clo
...
 
-* All SSH Internet Access to our application servers is required to be routed through our Company VPN which has a dedicated IP address.
-* New IAM roles are distributed for each use case and each service, preventing one user from having access to multiple parts of our internal system.
-* We require individual logins for every service that we use as a company. There are no shared accounts in the organization so we can effectively audit and log activity.
+* All access to any of our servers is required to be routed through our Corporate VPN that uses AES-256-CBC ciphers and SHA-256 encryption.
+* Unique IAM roles and permissions are distributed for each tier of our application infrastructure components, preventing one system from having access to others.
+* We require individual employee logins for every third-party service or platform that we use as a company. There are no shared accounts in the organization so we can effectively audit and log activity.
 * We require all passwords to be at least 12 characters long, randomly generated with with a mix of numbers, letters, capitalization, and symbols.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:59]: One too many "with"s.

File: background/keeping-your-data-secure.md [hunk 8]
```diff
@@ -53,14 +53,14 @@ When it comes to storing and organizing large datasets, we recommend using a clo
 
 Data security doesn't just end with our application. In the spirit of transparency, we also want you to know what our internal company policies are for accessing and handling sensitive data.
 
-* All SSH Internet Access to our application servers is required to be routed through our Company VPN which has a dedicated IP address.
-* New IAM roles are distributed for each use case and each service, preventing one user from having access to multiple parts of our internal system.
-* We require individual logins for every service that we use as a company. There are no shared accounts in the organization so we can effectively audit and log activity.
+* All access to any of our servers is required to be routed through our Corporate VPN that uses AES-256-CBC ciphers and SHA-256 encryption.
```
Comment (ericelsken) [inline background/keeping-your-data-secure.md:56]: Is this coming from OpenVPN? 
I know both of those things are used in the cipher suite, but it sounds weird saying "SHA-256 encryption", when that alone is not an encryption algorithm. AES is the encryption part.

File: background/keeping-your-data-secure.md (hunk 1 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:11]: Fixed.

File: background/keeping-your-data-secure.md (hunk 2 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:14]: Fixed.

File: background/keeping-your-data-secure.md (hunk 4 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:16]: For which line?

File: background/keeping-your-data-secure.md (hunk 5 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:29]: I dunno I think this was added by @blakeburch but I deleted that sentence just now.

File: background/keeping-your-data-secure.md (hunk 6 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:33]: Updated.

File: background/keeping-your-data-secure.md (hunk 7 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:59]: Fixed withs

File: background/keeping-your-data-secure.md (hunk 4 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:16]: Ok I think I updated the correct line.

File: background/keeping-your-data-secure.md (hunk 3 above)
Comment (basitshipyard) [reply background/keeping-your-data-secure.md:15]: RDS is using KMS, and KMS is using RSAES-OAEP-SHA-256 from my understanding - or will anyway once we switch to CMKs.

Source: https://docs.aws.amazon.com/kms/latest/developerguide/symm-asymm-choose.html

**Pull request merged**